pkg/redis/redis.go

```diff
@@ -0,0 +1,49 @@
+package redis
+
+import (
+	"errors"
+	"time"
+
+	"github.com/go-redis/redis"
+)
+
+type IRedis interface {
+	Initialise() *redis.Client
+	StoreKeyValue(client *redis.Client, key string, value string, expiration time.Duration)
+	GetKeyValue(client *redis.Client, key string) (string, error)
+	DeleteKey(client *redis.Client, key string) *redis.IntCmd
+}
+
+type ObjectStorageWrapper struct {
+}
+
+func (dw *ObjectStorageWrapper) Initialise() *redis.Client {
+	client := redis.NewClient(&redis.Options{
+		Addr:     "localhost:6379",
```

Comment:
make the configuration a value oin the `ObjectStorageWrapper` struct so that the entrypoint can pass it down.